cloudinary_cli/core/search.py

```diff
@@ -17,7 +17,8 @@
 e.g. cld search cat AND tags:kitten -s public_id desc -f context -f tags -n 10
 """)
 @argument("query", nargs=-1)
-@option("-f", "--with_field", multiple=True, help="Specify which asset attribute to include in the result.")
+@option("-f", "--with_field", multiple=True, help="Specify which asset attribute to add in the result.")
```

Comment:
What does multiple=True mean?  Is it that you can include this option more than once in the same command?

Reply:
yes, you can `--with_field tags --with_field context`, I also added support for combining them together, like `--with_field tags,context`

Reply:
Doesn't that end up being the same as fields then?